Automattic/pocket-casts-ios
# Onboarding: Various improvements

| 📘 Project: #500 |
|:---:|

This includes some various improvements and bug fixes:
- Adds back button to the onboarding views
- Updates the login header to:
  - Show larger images when SSO is disabled
  - Show random podcasts from the users database
- Styles the not now button
- Styles the back button for the plus landing
- Increases line spacing of the plus terms / privacy text
- No longer allows the user to swipe back to the plus purchase view
- Fixing some issues with the motion jumping
- Shows a close button if the plus landing view is the root view

## To test
1. Enable the FeatureFlag
2. Launch the app
3. Tap the Podcasts tab
4. Tap the Create Folder button
5. ✅ The login header looks good, the not now is styled
6. Tap the Log in button
7. ✅ Verify it has a back button that is white
8. Tap unlock all and subscribe
9. ✅ Verify you can't swipe back from the welcome view anymore
10. Tap on Import
11. ✅ Verify there is a back button
12. Tap on an import item
13. ✅ Verify there is a back button
14. Tap on Profile > Cog > Appearance
15. Select a theme that is locked
16. ✅ Verify you see the plus view with a close button

## Checklist
